src/test/compile-fail/issue-41394.rs

```diff
@@ -14,7 +14,7 @@ enum Foo {
 }
 
 enum Bar {
-    A = Foo::A as isize
+    A = Foo::A as isize //~ ERROR enums without inhabited variants do not have discriminants
```

Comment:
This makes the test no longer be a test of what it used to be testing. Could you add another variant before?